brands/amenity/cinema.json

```diff
@@ -165,6 +165,25 @@
       "name": "Cineplanet"
     }
   },
+  "amenity/cinema|Cineplex Entertainment": { 
```

Comment:
This should match the `name` tag..

Reply:
@bhousel resolved!

Reply:
It should be "Cineplex" right?   not "Cineplex Entertainment" ?

Reply:
@bhousel Cinepelx Entertainment is the official company name